### PR TITLE
[release-0.6] docs: mark release as unsupported

### PR DESCRIPTION
**Pull request by marquiz**
Refs #1131

### CODE REVIEW

**Comment by marquiz** [comment]
Adds a note with red background to the top of the page. No netlify preview but the output can be seen in my personal repo:
https://marquiz.github.io/node-feature-discovery/v0.6

File: docs/_includes/class/content-wrap.liquid [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        <button id="toggle" class="btn-octicon p-2 m-0 text-white" type="button">
+            <i class="fa fa-bars"></i>
+        </button>
+        <div class="title flex-1 d-flex flex-justify-center">
+            <a class="h4 no-underline py-1 px-2 rounded-1" href="{{ site.baseurl }}/">{{ site.title }}</a>
+        </div>
+    </div>
+    <div class="bg-red-2">
+        <div class="content p-3 px-sm-5">
+            This documentation is for NFD version that is no longer supported. Please upgrade and visit the
```
Comment (PiotrProkop) [inline docs/_includes/class/content-wrap.liquid:12]: s/NFD/Node feature discovery

**Review by PiotrProkop — COMMENTED** [review]
Nice HTML skills! Small nit not to use acronym in the warning.

File: docs/_includes/class/content-wrap.liquid (hunk 1 above)
Comment (marquiz) [reply docs/_includes/class/content-wrap.liquid]: Yeah, better. Changed as suggested

**Comment by marquiz** [comment]
> Nice HTML skills! Small nit not to use acronym in the warning.

No HTML skills required. This is basically just "forking" this one .liquid file from https://github.com/rundocs/jekyll-rtd-theme, adding this one <div/> banner there 😇

**Review by fmuyassarov — COMMENTED** [review]
/lgtm

**Pull request merged**